.github/workflows/ci.yaml

```diff
@@ -1,45 +1,45 @@
-# yamllint disable rule:line-length
-# yamllint disable rule:braces
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
 
 name: CI
 
 on:
-  pull_request:
+  pull_request: null
   push:
     branches:
-      - master
       - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Stop previous runs.

Reply:
I'd argue there's value in seeing previous builds finish to the end.